modules/administration/pages/disconnected-setup.adoc

```diff
@@ -33,7 +33,7 @@ We recommend you set up a dedicated RMT instance for each {productname} installa
 . On the RMT instance, install the RMT package:
 +
 ----
-zypper in rmt-server
+transactional-update pkg in rmt-server
```

Comment:
Do we ask the RMT server to be on a SLE Micro? I understand that the RMT / SMT server should be on a different machine, so there is no need to change from `zypper` to `transactional-update pkg`